docs/features/multiple-networks/multi-network-policies.md

```diff
@@ -1,20 +1,45 @@
 # MultiNetworkPolicies
-OVN-Kubernetes implements native support for
-[multi-networkpolicy](https://github.com/k8snetworkplumbingwg/multi-networkpolicy),
-an API providing
+
+## Introduction
+
+[multi-networkpolicy](https://github.com/k8snetworkplumbingwg/multi-networkpolicy) provides
 [network policy](https://kubernetes.io/docs/concepts/services-networking/network-policies/)
-features for secondary networks.
+features for secondary networks, and allow enhanced traffic security.
+
+## Motivation
+
+Using traffic restrictions based on user network configuration is a recommended practice. By defining policies that
+specify which services a workload can access, we mitigate supply chain attacks. Even if a component is compromised, it
+won’t be able to jeopardize other components of the system.
+
+## User-Stories
+
+As a cluster administrator, I want to enforce non-overridable network policies that must be adhered to by all user
+tenants in the cluster. This will secure my cluster’s network traffic.
```

Comment:
this is not a user story of network policies, since they apply per namespace. 

You're describing admin network policies, which we do not have for secondary networks.

Reply:
DONE wdyt?